plugins/modules/mysql_role.py

```diff
@@ -114,6 +114,13 @@
     type: bool
     default: no
 
+  members_must_exist:
```

Comment:
```suggestion
  fail_on_member:
```
Maybe? It's just an option from me. Let's also ask @bmalynovytch @rsicart @hubiongithub and @Jorge-Rodriguez  about naming

Reply:
Both names are well chosen, I like both. Using `fail_on_member` would be more uniform with postgresql, but `members_must_exist` sounds more explicit to me.

Reply:
cool, let's proceed with `members_must_exist` then:) thanks